api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/convolutional/Conv1D.kt

```diff
@@ -95,6 +95,8 @@ public class Conv1D(
         requireArraySize(dilations, 3, "dilations")
     }
 
+    /** Axis of height for which the extra dimension is added (unsqueezed) before actual
```

Comment:
Maybe better "Axis of height for which the extra dimension is added (unsqueezed) before actual convolution operation and the output from actual implementation are squeezed."

Reply:
thanks, fixed